### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by nesar** [nesar/AstroVAE#3]
![training_loss](https://user-images.githubusercontent.com/7386526/34658697-b56010d8-f3f6-11e7-8ad6-c9841b331650.png)

Not a lot of change with hyper-parameter changes (learning and decay rates).
